src/pgrepr/src/oid.rs

```diff
@@ -65,3 +65,6 @@ pub const FUNC_MZ_UPTIME_OID: u32 = 16_436;
 pub const FUNC_MZ_WORKERS_OID: u32 = 16_437;
 pub const TYPE_APD_OID: u32 = 16_438;
 pub const FUNC_LIST_EQ_OID: u32 = 16_439;
+pub const FUNC_MAX_APD_OID: u32 = 16_440;
+pub const FUNC_MIN_APD_OID: u32 = 16_441;
```

Comment:
I just remembered that the Apd-related OIDs are 17k+ - should we adjusts these here so that we fit into the normal numbering scheme? May be not worth it...

Reply:
Yeah those are all getting blown away in a few weeks anyway so shrug